docs/pages/config/+Page.mdx

```diff
@@ -295,7 +295,7 @@ Into:
 // Environment: config
 
 import type { Config } from 'vike/types'
-import Layout from '../layouts/LayoutDefault.jsx'// [!code --]
+//import Layout from '../layouts/LayoutDefault.jsx'// [!code --]
```

Comment:
Hm, that's quite unfortunate. It's really misleading for the user. Ideas for solutions?

Reply:
Or we only show JS.

Or we implement some kind of magic comment.

Reply:
For example:

```js
// @detype-temp-replace Layout Layout2
import Layout from '../layouts/LayoutDefault.jsx'// [!code --]
```

Reply:
Although the advantage of `@detype-temp-replace` is that detype will still apply.

But `//~` is maybe easier to implement.

As you wish.